doc/whats_new/v0.23.rst

```diff
@@ -14,6 +14,14 @@ Version 0.23.0
 
 Put the changes in their relevant module.
 
+New `n_features_in_` attribute
```

Comment:
Put this into the change log below, I think, under "Miscellaneous" or "multiple modules"? It does not concern most users.